packages/request-manager/tests/interceptor-whitelist.test.ts

```
@@ -135,6 +135,10 @@ describe('Interceptor', () => {
     });
 
     ['GET', 'POST', 'PUT', 'DELETE', 'PATCH'].forEach(method => {
+        beforeAll(() => {
+            jest.spyOn(console, 'error').mockImplementation(() => {});
+        });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeAll duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)